source/features/index.tsx

```diff
@@ -43,12 +43,12 @@ interface InternalRunConfig {
 
 let log: typeof console.log;
 
-function logError(id: FeatureID, error: Error | string, ...extras: unknown[]): void {
+function logError(id: FeatureID, error: Error | string | unknown, ...extras: unknown[]): void {
 	if (error instanceof TypeError && error.message === 'Object(...)(...) is null') {
 		error.message = 'The element wasn’t found, the selector needs to be updated.';
 	}
 
-	const message = typeof error === 'string' ? error : error.message;
+	const message = error instanceof Error ? error.message : error as string;
```

Comment:
If the code expects a string, make it a string
```suggestion
	const message = error instanceof Error ? error.message : String(error);
```